### PR TITLE
Update SRE posting for new reqs

### PR DESCRIPTION
**Pull request by CrashenX**

### CODE REVIEW

File: sre-operational-fabric.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ the world.
 ### Responsibilities
 
 - Help drive the architecture, develop, and maintain observability of customer environments
+- Help drive the architecture, develop, and maintain configuration maangement of customer environments
```
Comment (jedsmith) [inline sre-operational-fabric.md]: `maangement` -> `management`

Comment (jedsmith) [reply sre-operational-fabric.md]: Should we add a bullet back for observability? Not sure about losing that concept.

File: sre-operational-fabric.md [hunk 2]
```diff
@@ -90,6 +91,8 @@ The ideal candidate will have the following (**not** hard requirements):
 - Experience defining, developing, deploying, and operating systems that run at scale
 - Experience working on distributed teams
 - Experience with 0DD CI/CD on Kubernetes using Helm
+- Experience building declarative infrastructure (e.g. Terraform)
+- Experience instituting configuration management (e.g. Ansible Tower)
```
Comment (jedsmith) [inline sre-operational-fabric.md]: ", Puppet, Chef" will get more leads

**Review by jedsmith — CHANGES_REQUESTED** [review]
Xing for a typo, just general questions for the rest.

File: sre-operational-fabric.md (hunk 2 above)
Comment (CrashenX) [reply sre-operational-fabric.md]: kk

**Review by jedsmith — APPROVED** [review]

**Comment by jedsmith** [comment]
Thanks!

**Comment by neillc** [comment]
Coluld you please remove from this org.  I haven't worked for Rackspace for over a year and have requested removal multiple times.

**Pull request merged**